requestmanager/requestmanager.go

```diff
@@ -274,16 +274,15 @@ type processResponseMessage struct {
 	p         peer.ID
 	responses []gsmsg.GraphSyncResponse
 	blks      []blocks.Block
+	response  chan error
```

Comment:
I think this can be constrained to write-only: `response chan<- error`

Reply:
yea, probably. But the others aren't at the moment so I'd rather main the consistency for now